modules/account-lib/src/coin/eth/transactionBuilder.ts

```diff
@@ -415,7 +434,7 @@ export class TransactionBuilder extends BaseTransactionBuilder {
       .replace('0x', '');
     return walletSimpleByteCode + resultEncodedParameters;
   }
-  //endregion
+  // endregion

```

Comment:
these actually shouldn't be indented

https://youtrack.jetbrains.com/issue/IDEA-80636

Reply:
my intellij just did it on its own